docs/app/pages/components/components-sections/select/select-custom/snippets/app.html

```diff
@@ -0,0 +1,45 @@
+<div class="row">
+    <div class="col-md-5">
+
+        <ux-select-custom
+            [(selected)]="selected"
+            (filterChange)="filter = $event"
+            [allowNull]="allowNull"
+            [busy]="showBusyIndicator"
+            [width]="componentWidth"
+            [maxHeight]="componentMaxHeight"
+            [placeholder]="placeholder">
+
+            <ng-template #buttonContent>
+                <b>Selection:</b> {{ selected ? selected.name : '(none)' }}
+            </ng-template>
+
+            <ng-template #dropdownContent>
+
+                <div role="radiogroup"
```

Comment:
I'm a little torn here. The tabbable list will make the example easier, but the radio button list has the advantage that it shows all the implementation work that the user of the component (i.e. developer who incorporates the component) will have to consider. (If they insert a component that handles everything already, fine, but this example shows the "worst case".)

Reply:
I think if we are sticking to the example of radio buttons it might be better to use the tabbable list, as we would recommend the use of it for simple lists like this. It would also implement some of the additional accessibility features that the example is currently missing for you, such as only one item in the list should be tabbable at a time and if I click on an item it should be become the active tabbable item etc..

I agree that if someone was going to implement a more complicated component they may have to do this themselves, but we don't want to suggest that for a list like this someone should be implementing this manually when we have directives there for them to do this already.

Our demos are always limited in what we can show and will usually be much simpler than a real life example, but we aim to show the best practices for the example we are using which would be to use our well established and well tested directives.

Reply:
I have changed and committed this.